cmd/kpod/rmi.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/urfave/cli"
+)
+
+var (
+	rmiDescription = "Removes one or more locally stored images."
+	rmiCommand     = cli.Command{
+		Name:        "rmi",
+		Usage:       "Removes one or more images from local storage",
+		Description: rmiDescription,
+		Action:      rmiCmd,
+		ArgsUsage:   "IMAGE-NAME-OR-ID [...]",
+	}
+)
+
+func rmiCmd(c *cli.Context) error {
+	args := c.Args()
+	if len(args) == 0 {
+		return fmt.Errorf("container ID must be specified")
```

Comment:
Should this be container ID or image ID?  If tweaked here, I think buildah will need a similar tweak.